### PR TITLE
New version: Microsoft.NuGet version 6.8.0

### LINKED ISSUE #131541: [Update Request]: Microsoft.Nuget 6.8.0
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Microsoft.Nuget

### Package Version

6.8.0

### Please describe the changes you would like to see

new Nuget version 6.8.0 available at: https://dist.nuget.org/win-x86-commandline/v6.8.0/nuget.exe

### PR DESCRIPTION
**Pull request by BiosPlus**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:

- Resolves https://github.com/microsoft/winget-pkgs/issues/131541

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131544)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
The validation pipelines is currently under maintenance.

File: manifests/m/Microsoft/NuGet/6.8.0/Microsoft.NuGet.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.NuGet
+PackageVersion: 6.8.0
+InstallerType: portable
+Commands:
+- nuget
+AppsAndFeaturesEntries:
+- DisplayName: NuGet CLI
+  DisplayVersion: 6.8.0.127
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/NuGet/6.8.0/Microsoft.NuGet.installer.yaml:11]:
```suggestion
  DisplayVersion: 6.8.0.131
```
Version needs to be updated
image

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/NuGet/6.8.0/Microsoft.NuGet.installer.yaml (hunk 1 above)
Comment (BiosPlus) [reply manifests/m/Microsoft/NuGet/6.8.0/Microsoft.NuGet.installer.yaml]: Cheers! @mdanish-kh

**Review by mdanish-kh — APPROVED** [review]

**Comment by mdanish-kh** [comment]
> Azure Pipelines could not run because the pipeline triggers exclude this branch/path.



Probably that's why...

**Pull request merged**